data_gatherer.py

```diff
@@ -242,6 +258,10 @@ def main(argv):
     # Run the data-flow.
     node_handle = erdos.run_async()
 
+    pose_stream = None
```

Comment:
Please move this code, before the erdos.run_async() invocation. Preferably, close to where there log_gnss code is.